wsclient/src/main/java/org/hpccsystems/ws/client/platform/DebugValueInfo.java

```diff
@@ -0,0 +1,60 @@
+package org.hpccsystems.ws.client.platform;

```

Comment:
is this another candidate for the wrappers package?

Reply:
Yes, this and ApplicationValueInfo. This current PR's needed to allow hipie/salt/etc to use wsclient 7.0.2 though, and it's already pretty big. We can do a follow up PR that converts other info objects to wrappers.